main.go

```diff
@@ -123,7 +123,12 @@ func main() {
 	api.GET("/version", getVersionHandler)
 	api.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
 
-	_ = r.Run()
+	var bindAddress string
+	var bindAddressDefined bool
+	if bindAddress, bindAddressDefined = os.LookupEnv("BIND_ADDRESS"); !bindAddressDefined {
+		bindAddress = "0.0.0.0:8080"
+	}
+	_ = r.Run(bindAddress)
```

Comment:
Just want to also apply the same comment here as over at 
https://github.com/iver-wharf/wharf-provider-azuredevops/pull/11#discussion_r645333238 : 

I think it'd be better if this was extracted to its own function.

```go
func getBindAddress() string {
	bindAddress, ok := os.LookupEnv("BIND_ADDRESS")
	if !ok || bindAddress == "" {
		return "0.0.0.0:8080"
	}
	return bindAddress
}
```

Reply:
Done.